sdk/go.mod

```diff
@@ -3,33 +3,47 @@ module github.com/hashicorp/vault/sdk
 go 1.13
 
 require (
-	github.com/armon/go-metrics v0.3.0
-	github.com/armon/go-radix v0.0.0-20180808171621-7fddfc383310
-	github.com/aws/aws-sdk-go v1.25.37
+	github.com/armon/go-metrics v0.3.3
```

Comment:
Are the dep updates in here required by your changes? If not, it might be good to revert them, and address vendor updates separately.

Reply:
This could be due to a simple `go mod tidy`. If so, we should encourage that, right?

Reply:
Indeed I believe this is simply from `go mod tidy`. I'm inclined to leave as-is unless you object @calvn

Reply:
Makes sense, let's leave as-is.